tests/mocks/course_mocks.py

```diff
@@ -1,17 +1,12 @@
-mocked_subject_data = {"subjects": [{"subject": "CS", "descr": "Computer Science"}]}
-
 mocked_courses_data = {
-<<<<<<< HEAD
     "courses": [
         {
             "catalog_nbr": "0007",
             "descr": "INTRODUCTION TO COMPUTER PROGRAMMING",
             "crse_id": "105611",
         }
     ]
-=======
     "courses": [{"catalog_nbr": "0007", "descr": "INTRODUCTION TO COMPUTER PROGRAMMING", "crse_id": "105611"}]
```

Comment:
This merge conflict was resolved by including both versions rather than just one